public/app/features/dashboard/dashboard_model.ts

```diff
@@ -130,7 +136,7 @@ export class DashboardModel {
   }
 
   // cleans meta data and other non persistent state
-  getSaveModelClone() {
+  getSaveModelClone(saveVariables, saveTimerange) {
```

Comment:
I would maybe have one argument (called options?) instead of multiple for each setting. Maybe a matter of personal taste, but I think it makes it easier to introduce possible other settings in the future and handling defaults.

Here is one example of handling options with defaults: https://github.com/grafana/grafana/blob/master/public/app/core/config.ts#L25